ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/services/timebasedfragmentation/TimeBasedFragmentCreator.java

```diff
@@ -25,15 +24,16 @@ public class TimeBasedFragmentCreator implements FragmentCreator {
     private static final String TREE_GREATER_THAN_OR_EQUAL_TO_RELATION = "tree:GreaterThanOrEqualToRelation";
     private static final String TREE_LESSER_THAN_OR_EQUAL_TO_RELATION = "tree:LessThanOrEqualToRelation";
     private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+    private static final String GENERATED_AT_TIME = "generatedAtTime";
+    private static final String PROV_GENERATED_AT_TIME = "http://www.w3.org/ns/prov#"+GENERATED_AT_TIME;
```

Comment:
these values short not be hardcoded. It is a pure coincidence that we keep using prov#generatedAtTime.
Based on your current approach, should be added to TimeBasedConfig